roles/kubernetes-apps/ansible/templates/coredns-config.yml.j2

```diff
@@ -14,15 +14,15 @@ data:
         kubernetes {{ dns_domain }} in-addr.arpa ip6.arpa {
           pods insecure
 {% if resolvconf_mode == 'host_resolvconf' and upstream_dns_servers is defined and upstream_dns_servers|length > 0 %}
-          upstream {{ upstream_dns_servers|join(' ') }}
+          upstream {{ upstream_dns_servers.join(' ') }}
```

Comment:
I think this will break because Python lists don't have join method so you can't just replace filter with method call here.

Reply:
@eplightning We might have hit this bug.
We're checking this right now and might open a PR afterwards.

Reply:
#4470 fixes this